yarn.lock

```diff
@@ -15068,7 +15068,7 @@ encoding@^0.1.11:
   dependencies:
     iconv-lite "~0.4.13"
 
-end-of-stream@^1.0.0, end-of-stream@^1.1.0, end-of-stream@^1.4.1:
+end-of-stream@1.4.1, end-of-stream@^1.0.0, end-of-stream@^1.1.0, end-of-stream@^1.4.1:
```

Comment:
Mostly likely this also doesn't belong

Reply:
Do you understand why this is here - I think you were looking into it the other day?

Reply:
If you rebase, and re-run `yarn`, I think it might disappear.